releases/release-7.3.0.md

```diff
@@ -0,0 +1,305 @@
+
+    This feature provides an effective intervention method for sudden performance issues in the database. For performance issues caused by queries, before identifying the root cause, this feature can quickly alleviate its impact on overall performance, thereby improving system service quality.
+
+    For more information, see [documentation](/tidb-resource-control.md#query-watch-parameters).
+
+### SQL
+
+* List and List COLUMNS partitioned tables support default partitions [#20679](https://github.com/pingcap/tidb/issues/20679) @[mjonss](https://github.com/mjonss) @[bb7133](https://github.com/bb7133) **tw@qiancai** <!--1342-->
+
+    Before v7.3.0, when you use the `INSERT` statement to insert data into List or List COLUMNS partitioned tables, the data needs to meet the specified partitioning conditions of the table. If the data to be inserted does not meet any of these conditions, either the execution of the statement will fail or the non-compliant data will be ignored.
+
+   Starting from v7.3.0, List and List COLUMNS partitioned tables support default partitions. After a default partition is created, if the data to be inserted does not meet any partitioning condition, it will be written to the default partition. This feature improves the usability of List and List COLUMNS partitioning, avoiding the execution failure of the `INSERT` statement or data being ignored due to data that does not meet partitioning conditions.
+
+    Note that this feature is a TiDB extension to MySQL syntax. For a partitioned table with a default partition, the data in the table cannot be directly replicated to MySQL.
+
+    For more information, see [documentation](/partitioned-table.md#list-partition).
+
+### DB operations
+
+- note 1
+
+- note 2
+
+### Observability
+
+* Show the progress of collecting statistics [#44033](https://github.com/pingcap/tidb/issues/44033) @[hawkingrei](https://github.com/hawkingrei) **tw@Oreoxmt** <!--1380-->
+
+    Collecting statistics for large tables often takes a long time. In previous versions, you cannot see the progress of collecting statistics, and therefore cannot predict the completion time. TiDB v7.3.0 introduces a feature to show the progress of collecting statistics. You can view the overall workload, current progress, and estimated completion time for each subtask using the system table `mysql.analyze_jobs` or `SHOW ANALYZE STATUS`. In scenarios such as large-scale data import and SQL performance optimization, this feature helps you understand the overall task progress and improves the user experience.
```

Comment:
> In scenarios such as large-scale data import and SQL performance optimization, this feature helps you understand the overall task progress and improves the user experience.

Is this feature for viewing the progress of any background job, or just ANALYZE? If just ANALYZE, I'm not sure I understand the relevance of this quoted sentence.

Reply:
Just ANALYZE.  The common scenarios to issue ANALYZE manually: 
- After bulk load, ANALYZE the tables that are just loaded before providing the service
- When bad plan is found, it could be caused by stale statistics.  In this case, we will issue ANALYZE and wait for its completion. 

With progress of Analyze, users are able to know it's working and remaining seconds approximately. That's the reason I put this sentence.  Please help me refine the words if necessary.